docs/server_guide/state_management.rst

```diff
@@ -0,0 +1,10 @@
+########
+State Management
```

Comment:
On the rendering, the `State Management` section feels unnecessarily nested. It's a single indirection to the Implicit State Management page with no other info/links.

Maybe the State Management tab/page can also provide a link to the Sequence Batch Scheduler?